sixtyfps_runtime/corelib/items.rs

```diff
@@ -57,6 +57,22 @@ type ItemRendererRef<'a> = &'a mut dyn crate::item_rendering::ItemRenderer;
 pub type VoidArg = ();
 type KeyEventArg = (KeyEvent,);
 
+#[macro_export]
+macro_rules! declare_item_vtable {
+    (fn $getter:ident() -> $item_vtable_ty:ident for $item_ty:ty) => {
+        ItemVTable_static! {
+            /// The VTable for `$ty`
```

Comment:
the `$ty` will not work like this.  You need to use concat! in a #[doc] attribute.